src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -734,7 +737,12 @@ class UnifiedShareModal extends React.Component<Props, State> {
                     className="unified-share-modal"
                     isOpen={isConfirmModalOpen ? false : isOpen}
                     onRequestClose={submitting ? undefined : onRequestClose}
-                    title={<span>{this.renderModalTitle()}</span>}
+                    title={
```

Comment:
Might also be good to pull this out so it doesn't add more clutter to the title prop

Reply:
Maybe a new title component in its own file

Reply:
The reason for putting them together is to get the label rendered aligned, and avoid updating ModalDialog component in the even downstream. I put aside a method before passing them in the Modal. 
@priyajeet is that what you mean by "a new title component in its own file" ?

Reply:
A separate component in its own file if possible. The whole renderModalTitle() maybe can be moved there?